workshop/content/docs/advanced/security-access-control.md

```diff
@@ -4,4 +4,26 @@ title: Security and access control
 
 # Security and access control
 
-TODO
+Security is a very huge and complex topic and we do not pretend to cover it in this workshop. It affects the entire development lifecycle and it is recommended to follow security best practices during all the phases like design, development, deployment.
+We want to focus on API security and access control rather than the wide  topic about application security risks.
```

Comment:
rephrase like
*In this workshop we will focus only on API security and access control.*